app/services/candidate_interface/section_policy.rb

```diff
@@ -24,6 +24,15 @@ def personal_statement?
       @controller_path.classify.eql?('CandidateInterface::PersonalStatement')
     end
 
+    def work_history?
+      controllers = [
+        'CandidateInterface::RestructuredWorkHistory::Review',
+        'CandidateInterface::Volunteering::Review',
```

Comment:
Are we missing the work history breaks or is that covered in the RestructuredWorkHistory?

Reply:
Yes, it's included in `RestructuredWorkHistory`